ArduSub/Parameters.h

```diff
@@ -356,6 +356,9 @@ class ParametersG2 {
     AP_Proximity proximity;
 #endif
 
+    // beacon (non-GPS positioning) library
```

Comment:
this is not the only "non-GPS" system in subs. Is the beacon specific to UWB? If so, say that instead.